promslog/slog_test.go

```diff
@@ -94,6 +95,41 @@ func getLogEntryLevelCounts(s string, re *regexp.Regexp) map[string]int {
 	return counters
 }
 
+func TestDurationValues(t *testing.T) {
+	var (
+		buf    bytes.Buffer
+		output string
+		dur, _ = time.ParseDuration("1m30s")
+		config = &Config{Writer: &buf}
+	)
+
+	// Test logfmt.
+	logger := New(config)
+	logger.Info("duration testing", slog.Duration("duration_raw", dur), slog.String("duration_string", dur.String()))
+	output = buf.String()
+	// Print logs for humans to see, if needed.
+	fmt.Println(output)
```

Comment:
no need I think, `require` should print it when sth is wrong.

Reply:
True, we just do it for other tests currently, so it was a consistency thing -- certainly not required